src/site/content/en/fast/preload-critical-assets/index.md

```diff
@@ -137,6 +138,36 @@ import(_/* webpackPreload: true */_ "CriticalChunk")
+
+Preloading is a powerful performance optimization that has an effect on loading speed. Such optimizations can lead to changes in your site's [Core Web Vitals](/vitals/), and it's important to be aware them.
+
+### Largest Contentful Paint (LCP)
+
+Preloading has a powerful effect on [Largest Contentful Paint (LCP)](/lcp/) when it comes to fonts and images, as both images and text nodes can be [LCP candidates](/lcp/#what-elements-are-considered). Hero images and large runs of text that are rendered using web fonts can benefit significantly from a well-placed preload hint, and should be used when there are opportunities to deliver these important bits of content to the user faster.
+
+However, you want to be careful when it comes to preloading&mdash;and other optimizations! In particular, avoid preloading too many resources. When a resource is preloaded, its [resource priority](/prioritize-resources/) is effectively raised from whatever its initial priority would have been to "High" or "Highest". If too many resources are prioritized, effectively none of them are. The effects of excessive preload hints will be especially detrimental to those on slower networks where bandwidth contention will be more evident.
+
+Instead, focus on a few high-value resources that you know will benefit from a well-placed preload. When preloading fonts, ensure that you're serving fonts in WOFF 2.0 format to reduce resource load time as much as possible. Since WOFF 2.0 has [excellent browser support](https://caniuse.com/woff2), using older formats such as WOFF 1.0 or TrueType (TTF) will delay your LCP if the LCP candidate is a text node.
+
+When it comes to LCP and JavaScript, you'll want to ensure that you're sending complete markup from the server in order for the [browser's preload scanner](/preload-scanner/) to work properly. If you're serving up an experience that relies entirely on JavaScript to render markup and can't send server-rendered HTML, it may be advantageous to preload JavaScript critical for rendering in order to keep your LCP as low as possible.
+
+### Cumulative Layout Shift (CLS)
+
+[Cumulative Layout Shift (CLS)](/cls/) is an especially important metric where web fonts are concerned, and CLS has significant interplay with web fonts that use the [`font-display` CSS property](https://developer.mozilla.org/docs/Web/CSS/@font-face/font-display) to manage how fonts are loaded. To minimize web font-related layout shifts, consider the following strategies:
+
+1. **Preload fonts while using the default `block` value for `font-display`.** This is a delicate balance. Blocking the display of fonts without a fallback can be considered a user experience problem, however, fonts loaded with `font-display: block;` eliminates web font-related layout shifts. However, you still want to get those web fonts loaded as soon as possible if they're crucial to the user experience. Combining a preload with `font-display: block;` may be an acceptable compromise.
```

Comment:
You use "however" to begin in two back-to-back phrases. I think it may be worth rewording one of those.